### ISSUE TITLE
Does not create any swapfile

### ISSUE THREAD

**Issue by martin-ms** [Nefelim4ag/systemd-swap#113]
After starting the service on a ext3-filesystem, there is no attempt to create a swapfile, even if 

`swapfc_force_preallocated=1 # Will preallocate created files
`

has been set in the configuration. Instead of, the journal will be flooded with

`WARN: swapFC: ENOSPC`

messages every second.

**Comment by Jarel1337** [comment]
Does the latest master fix this issue?

**Comment by martin-ms** [comment]
Just did another pull of the latest master branch:

| systemd-swap start
| INFO: Load: /etc/systemd/swap.conf
| INFO: Zswap: backup current configuration: start
| INFO: Zswap: backup current configuration: complete
| INFO: Zswap: set new parameters: start
| INFO: Zswap: Enable: 1, Comp: lz4,  Max pool %: 25, Zpool: zbud
| INFO: Zswap: set new parameters: complete
| INFO: swapD: pickup devices from systemd-gpt-auto-generator
| INFO: swapD: searching swap devices
| WARN: swapFC: ENOSPC
| WARN: swapFC: ENOSPC
| WARN: swapFC: ENOSPC
| ...
| WARN: swapFC: ENOSPC

No changes, still does not create a swapfile and gives out endless warnings.

**Comment by Jarel1337** [comment]
And your filesystem is not full?

**Comment by martin-ms** [comment]
No, I have space left on root-device 
| Filesystem              Size  Used Avail Use% Mounted on
| dev                     1.4G     0  1.4G   0% /dev
| run                     1.5G  1.1M  1.5G   1% /run
| /dev/mapper/root         32G   26G  **3.9G**  87% /

and I am also able to manual activate swapspace
|               total        used        free      shared  buff/cache   available
| Mem:        2959528      886016      219620       84184     1853892     1805196
| Swap:       2959356           0     2959356

This space I have additional since I delete the existing swapfile before trying systemd-swap

**Comment by Jarel1337** [comment]
Has systemd-swap ever worked for you or was this regression introduced recently?
Also can you try running the following code where your swapfiles get created by swapfc (default `/var/lib/systemd-swap/swapfc/`):
```bash
chunk_size=$(numfmt --to=none --from=iec "256M") #Change this to what you have
path="/var/lib/systemd-swap/swapfc/" #Change this to what you have
FREE_BLOCKS=$(stat -f -c %f "${path}" )
echo $FREE_BLOCKS Free blocks
BLOCK_SIZE=$(stat -f -c %s "${path}")
FREE_BYTES=$(( FREE_BLOCKS * BLOCK_SIZE ))
echo $FREE_BYTES Free bytes
FREE_BYTES=$(( FREE_BYTES - chunk_size ))
echo $FREE_BYTES Free bytes
```

**Comment by aminvakil** [comment]
I faced this issue hours ago, that was becuase I forgot to set `swapfc_enabled=1`, maybe this?

**Comment by martin-ms** [comment]
It seems, that the message
`WARN: swapFC: ENOSPC`

comes from a misleading entry of my existing configuration file:
`swapfc_chunk_size=$(($RAM_SIZE))M      # Allocate size of swap chunk`

In the past, I set the size of swap chunk that way to get the same amount of built in memory as a swapfile. The syntax I got from the ZRam-block of the sample configuration file; and that worked.
Zhe syntax of the configuration file has been slightliy changes wit then latest release (at least at the ZRam-block) that the $-sign has been dropped, but it looks also that this format of chunk size ist no longer accepted. If I change it to a fix value like 
`swapfc_chunk_size=2890M     # Allocate size of swap chunk`

with this entry, the `WARN: swapFC: ENOSPC` disappeared and systemd-swap reports that a swapfile has been created. But now swapon refuses again to accept the swapfile since it has "probably holes". 
Can it be, that after the change of https://github.com/Nefelim4ag/systemd-swap/commit/c4df7cee876674c7a7a3b3f3d74d56d7a3826821 again "truncate" is involved on ext3-filesystems, and not "dd"?
